packages/templates/vite-template-web-viewer/src/components/Viewer.tsx

```diff
@@ -0,0 +1,123 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ECSchemaRpcInterface } from "@itwin/ecschema-rpcinterface-common";
```

Comment:
i dont think we need to import this ourselves in the template
the viewer should already be registering this under the hood as of 5.0 now